modules/gitops-release-notes-1-13-2.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-2_{context}"]
+= Release notes for {gitops-title} 1.13.2
+
+{gitops-title} 1.13.2 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.2_{context}"]
+== Errata updates
+
+[id="rhsa-XXXX:NNNN-gitops-1-13-2-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.13.2 security update advisory
+
+Issued: 2024-XX-YY
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-13-2_{context}"]
+== Fixed issues
```

Comment:
Shall we add RN for https://issues.redhat.com/browse/GITOPS-4234 and https://issues.redhat.com/browse/GITOPS-5640?

cc: @keithchong @jparsai

Reply:
If required, I can provide for [GITOPS-5640](https://issues.redhat.com/browse/GITOPS-5640)